docs/development.md

```diff
@@ -1,128 +1,119 @@
-# Garden React Development
+# Garden Development
 
-This is a multi-package repo which uses [Lerna](https://lernajs.io/) to
-manage shared and cross-package dependencies.
+You're here because you want to get into the codebase. That's great. This
+guide will help you dig into the sensational Garden dirt. If you haven't
+already, please see the [contribution](/.github/CONTRIBUTING.md)
+documentation for a project overview along with versioning, development, and
+PR workflows.
 
-For further contribution guidelines view our [contribution
-documentation](.github/CONTRIBUTING.md).
+## Package creation
 
-All packages must be implemented following the requirements listed below.
+One of the best ways to get familiar with Garden's package structure is to
+create your own. Garden makes this simple by providing a `yarn new` command.
+Upon running `yarn new` you'll be prompted to enter a name (e.g. `test`). The
+new package will be generated under `/packages` and you can view the results
+by running the `yarn start` command (e.g. `yarn start --scope @zendeskgarden/react-test`).
 
-## Creating New Packages
+Unless you are a member of the core team, it is unlikely that we'll accept
+brand new package PRs. However, a new package provides the simplified sandbox
+for exploring further component development concepts.
```

Comment:
Rather than just saying no, let's ask people to submit an issue for discussion. Something like

"If you feel that a new component or package is needed [open an issue.](#)"